docs/content/how-to/aws/disaster-recovery.md

```diff
@@ -1,17 +1,41 @@
 ---
-title: Migrating Hosted Cluster among the same AWS Region
+title: Disaster Recovery
 ---
 
-# Migrating Hosted Cluster among the same AWS Region
+# Disaster Recovery
 
-## Preface and Considerations
+## Migrating Hosted Cluster among the same AWS Region
+
+### Use cases for Disaster Recovery
+
+We want to help people to understand what situations could be considered a fit for this disaster recovery procedure.
+
+1. *The control-plane is down for your hosted cluster (api-server, etcd,...)*
+- For the first situation this procedure it's a good fit.
+
+2. *The management cluster API-server/etcd is down?*
+- For this second situation it does not make sense to use this procedure, so you can use a backup/recovery tool like Velero in order to recover the ETCD.
+
+3. *Hypershift operator is not working, it’s down and it cannot be recovered*
+- For this situation the procedure it's a good fit.
+
+!!! note Important
+    Other situations should be carefully dissected, to ensure the stability of the other deployed HostedClusters
```

Comment:
It's not clear what this is telling me. How am I going to dissect other situations? (also I'd prefer examine or inspect instead of dissect)